src/Components/Inquiry/__tests__/Views/InquiryBasicInfo.jest.tsx

```diff
@@ -89,16 +89,14 @@ describe("InquiryBasicInfo", () => {
     fill(wrapper, "profession", "Carpenter")
     fill(wrapper, "phone", "555-555-5555")
 
-    wrapper.find('[role="checkbox"]').first().simulate("click")
-
     wrapper.find("form").simulate("submit")
 
     await flushPromiseQueue()
 
     expect(mockSubmitUpdateMyUserProfile).toBeCalledWith({
       profession: "Carpenter",
       phone: "555-555-5555",
-      shareFollows: false,
+      shareFollows: true,
```

Comment:
**Q:** do you know why this is still being sent here?

Reply:
It is defined in the update profile mutation query in the inquiry flow, and when the user is created it defaults to `nil` [in Gravity](https://github.com/artsy/gravity/blob/main/app/models/domain/user.rb#L118), this way we make sure to set it to true when the user inquires about an artwork.

Another approach would be that we set the `share_follows` to `true` on Gravity when the user sends a message and remove it from Force.

But I'm open to other approaches as well :)